### PR TITLE
Setup automated deploy with Travis

### LINKED ISSUE #26: Add CI
And add pip deployment for tagged releases

### PR DESCRIPTION
**Pull request by Williangalvani**
This should deploy tags automatically.

Fix #26

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+deploy:
+  provider: pypi
+  user: bluerobotics
+  password:
+    secure: LBxDL7RlsRC0BLc+2PX7mdbI/BdCD7T/M7IOdLKpsgQ2GUYn2ONRps61bKAiOrl2k68weajD1E6Gl/lEoXtIF3NPt3JxRHlIBrpr3j+EPoG6sqKb5NahAoYX3j4qsNF17JMjTD+oT4EAnikwrSidHHbIt4RAYhtrKw0Br2Ep5MvaaFKOizAPV4bgPA5cni0IJ7C4nV7piGheLxgV6jCMyR9uBThDhqIuoZo8lnOdpwbc3/JMVKijAaJ27h5qpHCV9+SsfG7CYIA/vpafxBljYzcpJt3nKniA5JdPS/N+VXE0hvd9ejaMQY/o4goc8yFVoZFNCoogM7cP9731dzRT3YCJ/bv8hijlQTFU0JXKeHJQMySfpJRiy73k5fyQz5rkFvCfXy2ta2ej+T5aqd6jvJ/9lsHUPPIffPoIFDkwoLwBvkOwZ4oCNTVMqF95ey47asSm7HGragtdrR6sFht4M9xevDn1tndW8jK4bw4PRO7cljFnQLnUVH8eGCWQr8tiRvg9gawCezgHDeRGPVHJtnsN6JJDWWgxrBs5RLNog02AeRF48esVMY01ed5ti0gUCJlYYTrsqo1Zb356Ez/BZTEygRZhkTZ22RkVklbkAimsAeEGE/DL+umYq2PLza4NVL8KUMFuBiprRofuSh62H2xNEZ3miLxeXNxQdnSIEQ0=
```
Comment (patrickelectric) [inline .travis.yml]: just to make sure, did you create this secure key with travis secure variable tool ?

Comment (Williangalvani) [reply .travis.yml]: yep

**Comment by patrickelectric** [comment]
@Williangalvani  can you explain the "two version numbers in different places" ?

**Comment by Williangalvani** [comment]
@patrickelectric I meant we could match the tags against a regex, and if a tag "vX.y.Z" is founds, release a version X.y.Z, which would ignore the version number in setup.py"

**Comment by jaxxzer** [comment]
Can we add github releases also?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+install: true
+deploy:
+  provider: pypi
+  user: bluerobotics
+  distributions: sdist
+  password:
+    secure: hfzPRQJ03YyKvUrTChgj+CfduDuHf02OU3WREEMDsn35QH3KjHb7d1lGbIryo7+rR+qnoY7EsFAFkFYlTicQ8O0NbQuNhZjq/ls9aF7IFbkHqSVkKMotqtMc2M8kW2hmGs096Au2jZY1iSXHhkqBWvXa+Kyfmdin2hZcGgwzs7muMmSwk9/LwDucgGxsbLz33PgFUXGvMTRXMUPbQI9XDTe1q1GG7eCq1J12e0vZ9+Y0KJVeDtuyiCd0c9Vcf5TVpTdeMLyew/cMiN3xX32f7uyBJ+0KwBflI8qmEyT4xipVJ79c+lrg0CT3osTve+U4blbw5UNooxDgInQuxvXAYYO/7sqcUw+Bc5SRbOkres6KECXSv1p9aJMq+PRVnP6asu+IivwtHS3hw20/BruW6bTMP3HcA82fgLjq0xuL7g6ylPuLhT99PTUwcFzCTCBpg6s768Y2HR/p9q4sxbsde6fO+roldycO1Opsi2EelxuqbG8LCNYTn44xGNDm9WV4xU4m/ZDMD/1TBNcV381wJHoNkxOMQ7+ioqPJLpfbKV8bazaHAYfGhjXjlr7Tg76Qq5o6uma5A72XbcUOO8EUadlAdo2vNnWkbh30ohOj5930OCD63Ptdq5Kz1Te/ethL8yc1M3UEAU+mSfFkEv8deykW0OFbMVo2zAdpbKLsmnc=
+  on:
+    tags: true
+    condition: ~/ping-python/setup.py =~ $TRAVIS_TAG
```
Comment (patrickelectric) [inline .travis.yml]: A new line should be created in the end of the file.

**Comment by Williangalvani** [comment]
@jaxxzer Yes, we can, I'm looking into it

**Comment by patrickelectric** [comment]
> Can we add github releases also?

A new github release will be created with the new tag.

**Pull request merged**